docs/YAML_documents/yaml_structure.md

```diff
@@ -0,0 +1,48 @@
+YAML parser will receive a test yaml file as an input. 
+
+A test yaml file should have the following parameters:
+
+* test_id
+* object_size
+* part_size
+* range_read
+* part_range
+* min_runtime
+* sessions_per_node
+* sessions
+* tool
+
+---
+
+TEST_ID can be any unique string which can be treated as filtering through execution logs.  
+
+object_size & part_size can be given as range for test workload, so those can have the following 
```

Comment:
Also object_size & part_size can be fixed size value. e.g. in config/s3/s3api/multipart_fixed_object_size.yaml
Also object_size & part_size can be a list of values. e.g. in config/s3/s3api/copy_object_fix_size.yaml